addons/survey/static/tests/tours/survey_form.js

```diff
@@ -0,0 +1,268 @@
+        run: function () {}, // it's a check,
+    }, {
+        content: "Move Question 3 back below Question 2",
+        trigger: "tr.o_data_row:nth-child(2) td[name=sequence]",
+        run: "drag_and_drop_native div[name=question_and_page_ids] table tbody tr:nth-child(3)",
+    }, {
+        content: "Open that question again",
+        trigger: "tr.o_data_row td[data-tooltip='Question 3']",
+        run: "click",
+    },
+    ...changeTab("options"),
+    {
+        content: "Change trigger to confirm we can now use Question 2 again",
+        trigger: ".modal-content .o_field_widget[name=triggering_question_id] input",
+        run: "click",
+        in_modal: true,
+    }, {
+        content: "Question 2 is allowed as trigger again",
+        trigger: '.modal-content ul.ui-autocomplete a:contains("Question 2")',
+        in_modal: true,
+    }, {
+        content: "Change the trigger answer back as well",
+        trigger: ".modal-content .o_field_widget[name=triggering_answer_id] input",
+        in_modal: true,
+    }, {
+        content: "Set the second answer as trigger, then",
+        trigger: '.modal-content ul.ui-autocomplete a:contains("Answer B")',
+        run: "click",
+    },
+    ...saveAndClose(),
+    // delete this test survey
+    {
+        content: "Open Action menu to delete the test survey",
+        trigger: ".o_cp_action_menus button",
+    }, {
+        content: "Delete the test survey",
+        trigger: ".dropdown-menu span:contains('Delete')",
+        run: "click",
+    }, {
+        content: "Confirm deletion",
+        trigger: "button.btn-primary:contains('Ok')",
+        run: "click",
+    }
+
+]);
+
+function addTwoAnswers() {
+    return [
+        {
+            content: "Add the first answer",
+            trigger: "div[name=suggested_answer_ids] .o_field_x2many_list_row_add a",
+            in_modal: true,
+        }, {
+            trigger: 'tr.o_selected_row div[name=value] input',
+            run: 'text Answer A',
+            in_modal: true,
+        }, {
+            content: "Add the second answer",
+            trigger: "div[name=suggested_answer_ids] .o_field_x2many_list_row_add a",
+            in_modal: true,
+        }, {
+            trigger: 'tr.o_selected_row div[name=value] input',
+            run: 'text Answer B',
+            in_modal: true,
+        }
+    ];
+}
+
+function saveAndNew() {
+    return [
+        {
+            content: "Click Save & New",
+            trigger: "button.o_form_button_save_new",
+            in_modal: true,
+        }, {
+            content: "Wait for the dialog to render new question form",
+            // suggested_answer_ids required even though in_modal is specified...
+            trigger: "div[name=suggested_answer_ids] .o_list_table tbody tr:first-child:not(.o_data_row)", // empty answers list
+            in_modal: true,
+            run: function () {}, // it's a check,
+        }
+    ];
+}
```

Comment:
There is a "saveForm" method in the tools, but not sure it works as we have a modal here?

Reply:
It works, I dropped my "SaveAndClose".